netbox/templates/dcim/macaddress.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'generic/object.html' %}
+{% load helpers %}
+{% load plugins %}
+{% load render_table from django_tables2 %}
+{% load i18n %}
+
+{% block content %}
+  <div class="row">
+    <div class="col col-md-4">
+      <div class="card">
+        <h2 class="card-header">{% trans "MAC Address" %}</h2>
+        <table class="table table-hover attr-table">
```

Comment:
We should include the raw MAC address value as a row here, with a copy-to-clipboard button for convenience.